pkg/apis/abac/register.go

```diff
@@ -26,15 +26,27 @@ import (
 const Group = "abac.authorization.kubernetes.io"
 
 // Scheme is the default instance of runtime.Scheme to which types in the abac API group are registered.
+// TODO: remove this, abac should not have its own scheme.
```

Comment:
Federation might be ok, since you're doing something special. not sure about the scheduler.